src/components/organisms/AddSkilListSelector.tsx

```diff
@@ -42,6 +53,13 @@ const AddSkillListSelector = ({
     setOpenSkillDetails(true);
   };
 
+  /*
+   * QUERIES
+   */
+  const { data } = useQuery<SearchSkillsVerifiedQuery>(SEARCH_SKILLS_VERIFIED, {
+    variables: { search: `%${debouncedSearchValue}%` },
```

Comment:
```suggestion
    variables: { search: `%${search}%` },
```
Here you can use directly the `search` prop as useDebounce is already used in the parent component.

Reply:
This also avoid a delay in the refresh of result number in other categories